src/lib/meta/fn.ts

```diff
@@ -1,6 +1,8 @@
 /** noop */
-export const noop = () => {}
+export const noop = () => {};
 /** execute function */
-export const run = (fn: Function) => fn();
+export const run = (fn: () => any) => fn();
 /** execute array of functions */
-export const run_all = (fns: Function[]) => fns.forEach(run)
```

Comment:
I've used snake_case to delimit internal functions (no other reason than because Svelte does it the same way). Meta functions are currently not exposed. Also, many functions are now similar or the same as can be found in `svelte/internal`. This is a bigger discussion with two questions I have not decided on yet:
1. Do we want to expose meta utilities as general js utilities for the lib consumer?
2. If we don't expose them, can we use the svelte/internal fns instead? I think we can be quite confident that end-user will have svelte installed 😅 The only thing we would need to be careful of is that changes to svelte/internal are not advertised in the changelog.

Reply:
I am also a fan of defining internal functions with snake_case, so we can do that! But if we do, should all code that is not seen by the user be written in snake_case? This includes the code within function bodies that are public.

1. I guess we could, but it doesn't feel that important.
2. I think it's a better idea that we have our own meta instead of importing Svelte's internal functions. We don't know if they'll change/remove them for v4.

Reply:
> I am also a fan of defining internal functions with snake_case, so we can do that! But if we do, should all code that is not seen by the user be written in snake_case? This includes the code within function bodies that are public.

I think this is the way to go 👍 

> 1. I guess we could, but it doesn't feel that important.

Yeah, deliberating further I realise that I explicitly do _not_ aim to rival other general purpose utility libraries, there are better solutions for that (think lodash and the like) and doing that right would massively increase the scope of the project with stuff I'm not interested in. Less but better.

> 2. I think it's a better idea that we have our own meta instead of importing Svelte's internal functions. We don't know if they'll change/remove them for v4.

That's the thing, they can even change/remove between minor updates since they're internals. Let's keep em in like we have now 👍